models/commons/src/discounted-price/presets/index.ts

```diff
@@ -0,0 +1,3 @@
+export const restPresets = {};
+export const graphqlPresets = {};
+export const compatPresets = {};
```

Comment:
We don't need the `compatPresets` here since this is not a legacy migrated model but a new one.